keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -460,42 +410,47 @@ message PodSandboxAttributes {
 }
 
 // PodSandboxStats provides the resource usage statistics for a pod.
+// The linux or windows field will be populated depending on the platform.
 message PodSandboxStats {
     // Information of the pod.
-    // Corresponds to PodRef in SummaryAPI
     PodSandboxAttributes attributes = 1;
+    // Stats from linux.
+    LinuxPodSandboxStats linux = 2;
+    // Stats from windows.
+    WindowsPodSandboxStats windows = 3;
```

Comment:
just leaving note: This message design differs a bit from existing `message ContainerStats` in that we have sub messages for stats for windows/linux.  existing `message ContainerStats` does have separate submessages for windows/linux though. 

I prefer to have it this way though since it has more flexibility of allowing to extend stats for windows/linux separately.